src/osvr/Util/Logger.cpp

```diff
@@ -263,12 +266,12 @@ namespace util {
         }
 
         void Logger::flush() {
-            if (logger_) {
-                logger_->flush();
-            } else {
-                std::cout << std::flush;
-                std::cerr << std::flush;
-            }
+            logger_->flush();
+        }
+
+        void Logger::write(LogLevel level, const char* msg)
+        {
```

Comment:
We could regain the "absolute fallback" of `if (!logger_)` by just reverting the flush changes above and putting a conditional in this one function, now that we have all the stream handling entirely within our own codebase.